pkg/syncer/volume_health.go

```diff
@@ -111,7 +111,7 @@ func csiGetVolumeHealthStatus(ctx context.Context, k8sclient clientset.Interface
 									newPvc.Namespace, newPvc.Name, val, volHealthStatus)
 								metav1.SetMetaDataAnnotation(&newPvc.ObjectMeta, annVolumeHealth, volHealthStatus)
 								metav1.SetMetaDataAnnotation(&newPvc.ObjectMeta, annVolumeHealthTS, time.Now().Format(time.UnixDate))
-								log.Infof("set annotation for health to %s at time %s", volHealthStatus, time.Now().Format(time.UnixDate))
+								log.Infof("csiGetVolumeHealthStatus: set annotation for health to %s at time %s for pvc %s/%s ", volHealthStatus, time.Now().Format(time.UnixDate), newPvc.Namespace, newPvc.Name)
```

Comment:
Edit the log on line 100 as well to keep it consistent. Also, we already have a debug log capturing the same info(on line 109) . So do we even need this log?